virt/virt-2-4-release-notes.adoc

```diff
@@ -182,19 +182,9 @@ value by running `oc describe node <node>` for all nodes and looking at the
 `cpu-model-<name>` labels. Select the CPU model that is present on all of your
 nodes.
 
-* The {VirtProductName} upgrade process occasionally fails due to an interruption
-from the Operator Lifecycle Manager (OLM). This issue is caused by the limitations
-associated with using a declarative API to track the state of {VirtProductName}
-Operators. Enabling automatic updates during
-xref:../virt/install/installing-virt-web.adoc#virt-subscribing-to-the-catalog_installing-virt-web[installation]
-decreases the risk of encountering this issue.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1759612[*BZ#1759612*])
-
 //Jira story for the following content https://issues.redhat.com/browse/CNV-4757. Nelli noted that for CNV 2.5, this can be removed from Known Issues. The defect status, however, is CLOSED.
 * {VirtProductName} cannot reliably identify node drains that are triggered by
 running either `oc adm drain` or `kubectl drain`. Do not run these commands on
 the nodes of any clusters where {VirtProductName} is deployed. The nodes might not
 drain if there are virtual machines running on top of them.
-The current solution is to put nodes into maintenance.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
-
+The current solution is to xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc[put nodes into maintenance].
```

Comment:
Looks like you're off one level here and missing the anchor. Try `xref:../../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-maintenance`

Reply:
Added the anchor, thanks.